app/src/main/java/org/fossasia/openevent/app/event/attendees/AttendeesFragment.java

```diff
@@ -148,10 +147,8 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
             return;
         long attendeeId = data.getLongExtra(Constants.SCANNED_ATTENDEE, -1);
 
-        if (attendeeId != -1) {
-            attendeesPresenter.getAttendeeById(attendeeId)
-                .subscribe(this::showToggleDialog, Timber::e);
-        }
+        if (attendeeId != -1)
```

Comment:
Now as we got this dialog fragment we shall show this dialog on QR scan activity itself no need to start it for result.

What do you say? 
don't have to do in this PR itself. I will open issue for the same.